var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -29,6 +29,9 @@ class Libxc(AutotoolsPackage, CudaPackage):
     patch('0001-Bugfix-avoid-implicit-pointer-cast-to-make-libxc-com.patch', when='@5.0.0')
     patch('0002-Mark-xc_erfcx-a-GPU_FUNCTION.patch', when='@5.0.0')
 
+    patch('nvhpc-configure.patch', when='%nvhpc')
+    patch('nvhpc-libtool.patch', when='%nvhpc@develop')
```

Comment:
What does `%nvhpc@develop` actually mean? Will Spack ever detect `%nvhpc` as version `develop`?

Reply:
Sorry, it should be `@develop %nvhpc`.